src/reducers/utxo_by_address.rs

```diff
@@ -23,6 +24,16 @@ impl Reducer {
         tx_idx: usize,
         output: &mut super::OutputPort,
     ) -> Result<(), gasket::error::Error> {
+        if self.config.filter.is_some() {
+            match self.config.filter.as_ref().unwrap().binary_search(&address.to_string()) {
+                Ok(_) => {
+                    log::info!("Address '{}' added with UTXO data", address);
```

Comment:
Using `info` log level seems a little excessive. I would use `debug` level, if any.

Ideally, we should promote metrics / counters rather than logs for these high-frequency observations. Not required for this PR, but it might be a nice addition for a follow up.